docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -399,7 +411,7 @@ const rows = apiRef.current.getRowGroupChildren({
 {{"demo": "RowGroupingGetRowGroupChildren.js", "bg": "inline", "defaultCodeOpen": false}}
 
 :::warning
-The `apiRef.current.getRowGroupChildren` method is not compatible with the [server-side row grouping](/x/react-data-grid/server-side-data/row-grouping/) since all the rows might not be available to get at a given instance.
+The `apiRef.current.getRowGroupChildren` method is not compatible with [server-side row grouping](/x/react-data-grid/server-side-data/row-grouping/) because not all rows may be available to retrieve in any given instance.
```

Comment:
```suggestion
The `apiRef.current.getRowGroupChildren()` method is not compatible with [Server-side data-Row grouping](/x/react-data-grid/server-side-data/row-grouping/) because not all rows may be available to retrieve in any given instance.
```